script/e2e-test.sh

```diff
@@ -198,6 +198,7 @@ installOrUpgradeKubeapps() {
     --set apprepository.initialRepos[0].url=http://chartmuseum-chartmuseum.kubeapps:8080
     --set apprepository.initialRepos[0].basicAuth.user=admin
     --set apprepository.initialRepos[0].basicAuth.password=password
+    --set featureFlags.operators=true
```

Comment:
We will probably need to have a switch (env var) for CI so that we can run CI with or without the operator support and tests, but doesn't need to be done here and now, just noting.

Reply:
Absolutely, by now we will have CI running with operators to check that we keep on supporting it 😄

Reply:
Wow, only now I see your comments below, forget my comment above this one 😃